src/screens/Settings/AccountSettings.tsx

```diff
@@ -97,7 +102,11 @@ export function AccountSettingsScreen({}: Props) {
           )}
           <SettingsList.PressableItem
             label={_(msg`Change email`)}
```

Comment:
```suggestion
            label={_(msg`Update email`)}
```
I suggest using `Update` here to be consistent with the language used in other strings.